src/libsync/account.h

```diff
@@ -172,6 +172,7 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject {
 
     void clearCookieJar();
     void lendCookieJarTo(QNetworkAccessManager *guest);
+    QString cookieJarPath();
```

Comment:
It's accessed from 2 different files/classes which are not the account, can't do it without friending both.